ChatQnA/tests/test_chatqna_on_xeon.sh

```diff
@@ -143,6 +168,8 @@ function main() {
     validate_microservices
     validate_megaservice
 
+    run_e2e_tests
```

Comment:
use `validate_ui` or `validate_frontend`.  
`e2e` including backend and frontend.

Reply:
fixed